02-wrangling.Rmd

```diff
@@ -2,14 +2,14 @@
 
 ## Overview 
 
-This chapter will be centered around tools for cleaning and wrangling data that move data from its raw format into a format that is suitable for data analysis. They 
-will be presented in the context of a real world data science application, providing more practice working through a whole case study.
+This chapter will be centred around tools for cleaning and wrangling data that move data from its raw format into a suitable format for data analysis. They will be presented in the context of a real-world data science application, providing more practice working through a whole case study.
```

Comment:
```suggestion
This chapter will be centred around defining tidy data, 
a data format that is suitable analysis, 
and the tools needed to transform raw data into this format.
This will be presented in the context of a real-world data science application, 
providing more practice working through a whole case study.
```